backend/tests/api/support/index.js

```diff
@@ -0,0 +1,144 @@
+const request = require('supertest');
+const { jwt } = require("../../helpers/strapi");
+const userFactory = require("../../user/factory");
+
+describe("Support Test", () => {
+    let authenticatedUser;
+    let adminUser;
+    beforeAll(async () => {
+        console.log("Support before all");
+        authenticatedUser = await userFactory.createUser(strapi, "authenticated");
+        adminUser = await userFactory.createUser(strapi, "admin");
+    });
+
+    /**
+     * @param {string | number} id
+     */
+    const constructSupportTicket = (id) => {
+        return {
+            data: {
+                "FullName": "Test Name" + id,
+                "Email": "test" + id + "@email.com",
+                "Phone": "111-111-1111",
+                "Title": "Test Title" + id,
+                "Message": "Test Message" + id,
+                "DateSubmitted": Date.now(),
+                "Status": "Submitted", // "Under Review" or "Completed"
+                "Memo": "Test Memo" + id,
+                "DateCompleted": Date.now(),
```

Comment:
When a support ticket is created, is there DateCompleted?

Reply:
Made null